libs/platform/src/lib/components/menu/menu-component/menu-item.component.scss

```diff
@@ -0,0 +1,104 @@
+.fd-menu__item {
+    &:hover {
+        //don't apply hover properties on only label item, we need it over entire item
+        background-color: transparent;
+    }
+    text-overflow: ellipsis;
+    min-width: 192px;
+    height: 40px; //line=24 px. calc is 40-8-8(padding top/bottom)=24px
+    overflow: hidden;
+    background-color: transparent;
+    padding: 12px 16px;
+    color: var(--fd-menu-item-color);
```

Comment:
When using CSS variables, we need to also assign a hardcoded default color, since IE11 doesn't recognize the `var()` syntax.

So change:
```css
   color: var(--fd-menu-item-color);
```

to:

```scss
   color: #51555a; 
   color: var(--fd-menu-item-color);
```

Be sure to add the "hardcoded" value before the `var()` value. For browsers that recognize `var()`, the hardcoded value will be overwritten.

Reply:
done